README.md

```diff
@@ -17,20 +19,20 @@ development-related projects are now becoming larger and more complex and they n
 intermediate software engineering skills to help them design more robust software code, 
 automate the process of testing and verifying its correctness and support collaborations with others.
 
-## About the Lesson
-
-This lesson teaches intermediate-level software development skills in a way that mimics a typical software development 
-process in a team, starting from an existing piece of software.
-
-The software project template repository used for this lesson is available at:
-<https://github.com/carpentries-incubator/python-intermediate-inflammation>.
-
 ### Lesson Status
-The lesson is currently in beta - it has been tested several times with 
-different cohorts by the lesson authors as well as independently by people not involved in the course development
+
+The lesson is currently in beta - it has been over 10 times times with 
+different cohorts by the lesson authors as well as independently by people not directly involved in the lesson development
 and is in a good state to be reused and taught by others.
 
 ## Teaching the Lesson
+
+The lesson has 5 sections that can be delivered (approximately) over 5 half-days. The lesson is suitable either for 
```

Comment:
```suggestion
The lesson has 5 sections; each section can be delivered or worked through in self-learning mode over a half a day to a day depending on the pace (we would recommend setting aside 1 day per section to make sure the delivery is not rushed). The lesson is suitable either for 
```

Reply:
How about this for an estimate in time needed @douglowe ?

Reply:
that works for me :)